src/modules/ArticlesServices.js

```diff
@@ -13,12 +18,14 @@ const ArticlesServices = {
             lead: event.target.lead.value,
             body: event.target.body.value,
             category_id: parseInt(event.target.categories.value),
+            image: encodedImage,
           },
         },
         {
           headers: headers,
         }
       );
+      event.target.reset();
```

Comment:
Is this for cleaning the inputs?

Reply:
Yes! It will reset the input fields after creating an article.